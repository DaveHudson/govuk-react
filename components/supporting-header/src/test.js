import React from 'react';
import { mount } from 'enzyme';

import SupportingHeader, { exampleHeading } from './fixtures';

describe('SupportingHeader', () => {
  let wrapper;

  it('renders without crashing', () => {
    wrapper = mount(<SupportingHeader>{exampleHeading}</SupportingHeader>);
  });

  it('matches wrapper snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
