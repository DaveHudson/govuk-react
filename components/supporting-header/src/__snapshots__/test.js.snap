// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SupportingHeader matches wrapper snapshot 1`] = `
.emotion-1 {
  margin-bottom: 0;
}

@media only screen and (min-width:641px) {
  .emotion-1 {
    margin-bottom: 0;
  }
}

.emotion-0 {
  font-family: "nta",Arial,sans-serif;
  font-size: 20px;
  color: #6f777b;
  margin-bottom: 0;
}

@media only screen and (min-width:641px) {
  .emotion-0 {
    font-size: 27px;
  }
}

@media only screen and (min-width:641px) {
  .emotion-0 {
    margin-bottom: 0;
  }
}

<Styled(SupportingHeader)>
  <SupportingHeader
    className="emotion-1"
  >
    <Styled(span)
      className="emotion-1"
    >
      <span
        className="emotion-0"
      >
        Heading text
      </span>
    </Styled(span)>
  </SupportingHeader>
</Styled(SupportingHeader)>
`;
